Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -129,7 +129,7 @@ WebIDL::ExceptionOr<JS::Value> XMLHttpRequest::response()
     if (m_response_type == Bindings::XMLHttpRequestResponseType::Empty || m_response_type == Bindings::XMLHttpRequestResponseType::Text) {
         // 1. If this’s state is not loading or done, then return the empty string.
         if (m_state != State::Loading && m_state != State::Done)
-            return JS::PrimitiveString::create(vm, "");
+            return JS::PrimitiveString::create(vm, String {});
```

Comment:
this belongs into the first commit, not last

Reply:
sure